reconcile/statuspage/integrations/maintenances.py

```diff
@@ -33,8 +33,18 @@ def reconcile(
         diff = diff_iterables(
             current=current_state, desired=desired_state, key=lambda x: x.name
         )
+        errors = False
         for a in diff.add.values():
-            logging.info(f"Create StatusPage Maintenance: {a.name}")
+            if not a.components:
```

Comment:
no `components` case is checked [here](https://github.com/app-sre/qontract-reconcile/blob/a18009361b26a8b1fe5b07b8a93419f79bc87136/reconcile/statuspage/page.py#L157), can we validate it in the same place?

Reply:
Updated. Now if there are no attached component, we cannot set a status page maintenance (which makes sense IMHO)